includes/frontend/class-wcmp-checkout.php

```diff
@@ -245,6 +245,26 @@ public function getDeliveryOptionsConfig(): array
                     $value += $chosenShippingMethodPrice;
                 }
 
+                if(in_array($settingName, [WCMYPA_Settings::SETTING_CARRIER_DELIVERY_MORNING_ENABLED, WCMYPA_Settings::SETTING_CARRIER_DELIVERY_EVENING_ENABLED], true)) {
+                    $ageCheckFromSettings = (bool) WCMYPA()->setting_collection->getByName(sprintf('%s_%s', $carrier, WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_AGE_CHECK));
+                    $ageCheckFromProduct = false;
+                    foreach (WC()->cart->get_cart() as $cartItem) {
+                        /**
+                         * @var WC_Product $product
+                         */
+                        $product = $cartItem['data'];
+                        $ageCheckMeta = $product->get_meta(WCMYPA_Admin::META_AGE_CHECK, true);
+                        if($ageCheckMeta === 'yes') {
```

Comment:
is dit nu niet hetzelfde als een bestaand stuk code?

Reply:
Ik heb iets vergelijkbaars gezien met `WCX_Product` class, dat in een order zat. Dit is nog geen order dus leek voor mij niet herbruikbaar in deze context.